todo/dto/label_dto.py

```diff
@@ -0,0 +1,13 @@
+from datetime import datetime
+from pydantic import BaseModel
+
+from todo.dto.user_dto import UserDTO
+
+
+class LabelDTO(BaseModel):
+    name: str
+    color: str
+    createdAt: datetime | None = None
+    updatedAt: datetime | None = None
+    createdBy: UserDTO | None = None
+    updatedBy: UserDTO | None = None
```

Comment:
These values can ever be none?

Reply:
Yes. This label DTO can be used in other places also
In get tasks API, we only need to send name and color of the label